src/main.tsx

```diff
@@ -1,24 +1,31 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import App from './App.tsx'
-import {
-  createBrowserRouter,
-  RouterProvider,
-} from "react-router-dom";
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import App from "./App.tsx";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import ErrorPage from "./error-page";
-import './index.css'
+import "./index.css";
+import { ThemeProvider } from "styled-components";
```

Comment:
More automatic formatting by Prettier.  Seems to like double quotes, trailing commas, and semicolons